polars_queries/q03.py

```diff
@@ -0,0 +1,37 @@
+import polars as pl
+from linetimer import linetimer
+
+from polars_queries import datasets
+
+
+@linetimer(name="Query 03", unit='s')
+def q03():
+    customer_ds = (datasets.get_customer_ds()
+                   .filter(pl.col("c_mktsegment") == "HOUSEHOLD")
+                   .select(["c_custkey", "c_mktsegment"])
```

Comment:
This is immediately a proper test for the optimizer.

You should not have to define this `selection`. Polars should be able to define which columns to select by itself.

So removing these lines should be as fast as with these lines.

Reply:
> for this, it seems like my brain optimizer ran before the polars optimizer

LOL

> same except for 1 additional FAST PROJECTION in my version

That should be fine.